Samples/CSharp/stylecop.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
+  "settings": {
+    "documentationRules": {
+      "copyrightText": "Copyright (c) 2019 Henry Gabryjelski",
```

Comment:
`.editorconfig` was recently updated to provide header comments too. Although, I've never tried it personally.

Reply:
More info here:

https://docs.microsoft.com/en-us/visualstudio/ide/reference/add-file-header?view=vs-2019

Perhaps we should add this and comment it out by default?

Reply:
sounds like a good improvement.